project/src/main/java/com/google/sps/utilities/SpannerTasks.java

```diff
@@ -112,7 +112,7 @@ public static void insertOrUpdateUser(User user) {
    *
    * @param event the event to be inserted or updated; event's ID field should not exist in DB
    */
-  public static void insertorUpdateEvent(Event event) {
+  public static void insertOrUpdateEvent(Event event) {
     List<Mutation> mutations =
```

Comment:
Please revert this change I wrote tests with the spelling insertorUpdate so those will fail.